README.md

```diff
@@ -40,12 +40,23 @@ domain we would be very interested in supporting it.  Please
 suggestion.  PRs are always welcome and greatly appreciated, but for larger functional changes a pre-coding introduction
 can be helpful to ensure this is the correct place and that active or conflicting efforts don't exist.
 
-## Owners
+Approvers ([@open-telemetry/java-instrumentation-approvers](https://github.com/orgs/open-telemetry/teams/java-instrumentation-approvers)):
 
-- [Anuraag Agrawal](https://github.com/anuraaga), AWS
+- [Lauri Tulmin](https://github.com/laurit), Splunk
 - [Pablo Collins](https://github.com/pmcollins), Splunk
-- [Ryan Fitzpatrick](https://github.com/rmfitzpatrick), Splunk (maintainer)
+
+Maintainers ([@open-telemetry/java-instrumentation-maintainers](https://github.com/orgs/open-telemetry/teams/java-instrumentation-maintainers)):
```

Comment:
Hmm, does java-contrib have its own approver/maintainer groups? Should we reuse the instrumentation repo ones herE?

Reply:
Contrib has its own, but adding the instrumentation team as a member to it seems to make sense to me.

Reply:
oops this was just copy-paste, fixed

I ran into problem with using sub-teams, where the members of the sub-teams couldn't invite new people to the parent team, which I later found out was captured by:

https://github.com/open-telemetry/community/blob/main/docs/how-to-configure-new-repository.md#permissions

> 2. Every member of foo-maintainers should be included in foo-approvers and foo-triagers explicitly, with the "Maintainer" GitHub privileges. This allows repository maintainers to invite new approvers and triagers to the team.